Makefile

```diff
@@ -40,3 +40,5 @@ mutations: ## Check code for mutants
 		--coverage=../var/coverage \
 		--skip-initial-tests \
 		$(ARGS)
+bench:
```

Comment:
`bench: ## Performance benchmark`
- type `make` it lists all the by comment documented commands :)